api/README.md

```diff
@@ -236,3 +236,27 @@ For additional information, see jasypt spring boot [documentation](https://githu
 
 Tip: If using GitLab CI Runner, specify the value for JASYPT_ENCRYPTOR_PASSWORD as a secure variable. Secure variables can be added to a Gitlab project by navigating to Project Settings > Variables > Add Variable. 
 A secure variable's value is by default not visible in the build log and can only be configured by an administrator of a project.
+
+### Securing APIs Basic Auth
+
+From the admin menu, generate an "apitoken" for an "apiuser"
+
+Create a POST request with the following 2 headers and retrieve a JWT token for secured API calls.
```

Comment:
@rob-miller-777 There are two tokens. The first one is the apitoken that the system uses to authenticate. Hygieia sends back a JWT token in response with ROLE_API authority. The calling system then uses the JWT token to make its calls.